components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -38,6 +39,7 @@ data class BrowserState(
     val containers: Map<String, ContainerState> = emptyMap(),
     val extensions: Map<String, WebExtensionState> = emptyMap(),
     val activeWebExtensionTabId: String? = null,
+    val inProgressMediaTabId: String? = null,
```

Comment:
I think we will need a more flexible approach here. Let's model this differently and instead add a `lastMediaAccess` (or `lastMediaPlayed`) timestamp to `TabSessionState`, similar to https://github.com/mozilla-mobile/android-components/blob/master/components/browser/state/src/main/java/mozilla/components/browser/state/state/TabSessionState.kt#L40

This way we could for example show a sorted list of recent media tabs instead of just one.

Reply:
(To show the most recent tab that played media we can just look at the list of tabs and take the one with the newest `lastMediaAccess` then.)